rc/trafficserver.service.in

```diff
@@ -17,7 +17,7 @@
 #
 [Unit]
 Description=Apache Traffic Server is a fast, scalable and extensible caching proxy server.
-After=syslog.target network.target
+After=network.target
```

Comment:
`syslog.target` is [removed from recent versions of systemd](https://github.com/systemd/systemd/blob/1cee1c52833fb6e3829e510109404852a17e5bdd/NEWS#L8931).